src/code_fee30_len_2d60.c

```diff
@@ -18,9 +18,26 @@ INCLUDE_ASM("code_fee30_len_2d60", spr_transform_point);
 
 INCLUDE_ASM("code_fee30_len_2d60", spr_draw_component);
 
-INCLUDE_ASM("code_fee30_len_2d60", _spr_sign_extend_12bit);
-
-INCLUDE_ASM("code_fee30_len_2d60", _spr_sign_extend_16bit);
+s32 _spr_sign_extend_12bit(u16 val) {
+    s32 temp = val & 0xFFF;
+
+    if ((temp & 0x800)) {
+        return temp | -0x1000;
+    } else {
+        return temp;
+    }
+}
+
+//INCLUDE_ASM("code_fee30_len_2d60", _spr_sign_extend_16bit);
```

Comment:
don't forget to remove the comment here

Reply:
Was done locally but I forgot to push I guess lol